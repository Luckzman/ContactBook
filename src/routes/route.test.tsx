import React from 'react';
import { shallow } from 'enzyme';
import MainRoute from './';

it('renders without crashing', () => {
  const wrapper = shallow(<MainRoute />);
  expect(wrapper).toMatchSnapshot();
});
