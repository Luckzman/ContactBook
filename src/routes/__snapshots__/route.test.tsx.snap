// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      key="route0"
      path="/"
    />
    <Route
      component={[Function]}
      exact={false}
      key="route1"
      path="*"
    />
  </Switch>
</BrowserRouter>
`;
